projects/worldskills-angular-lib/package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "@worldskills/worldskills-angular-lib",
-  "version": "9.1.0",
-  "description": "A library to help speed up the building of worldskills angular applications",
+  "name": "worldskills-angular-lib",
```

Comment:
Shouldn't the name stay `@worldskills/worldskills-angular-lib` as per npm repo? https://www.npmjs.com/package/@worldskills/worldskills-angular-lib